MediaStreamExt::error() requires a fully-qualified call now

I updated to latest gtk4-rs: https://gitlab.gnome.org/YaLTeR/video-trimmer/-/commit/83e0f4015484ad3c566db2d11fa9f126a12cb65c#596ffd44ab388aa7cc024f7219a9c39f7788215f

Generally everything became better, except this one change I had to do: https://gitlab.gnome.org/YaLTeR/video-trimmer/-/commit/83e0f4015484ad3c566db2d11fa9f126a12cb65c#596ffd44ab388aa7cc024f7219a9c39f7788215f_153_147

```diff
-let error = media_file.get_error().unwrap();
+let error = MediaStreamExt::error(media_file).unwrap();
```

Because `error()` is also found in a few other associated traits. I wonder why it worked before with `get_error()` then?

```
error[E0034]: multiple applicable items in scope
   --> src/video_preview.rs:138:44
    |
138 |                     let error = media_file.error().unwrap();
    |                                            ^^^^^ multiple `error` found
    |
    = note: candidate #1 is defined in an impl of the trait `gtk4::prelude::GLAreaExt` for the type `O`
    = note: candidate #2 is defined in an impl of the trait `gtk4::prelude::MediaStreamExt` for the type `O`
    = note: candidate #3 is defined in an impl of the trait `gtk4::prelude::PrintOperationExt` for the type `O`
    = note: candidate #4 is defined in an impl of the trait `gtk4::prelude::MediaStreamExtManual` for the type `O`
help: disambiguate the associated function for candidate #1
    |
138 |                     let error = gtk4::prelude::GLAreaExt::error(&media_file).unwrap();
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: disambiguate the associated function for candidate #2
    |
138 |                     let error = gtk4::prelude::MediaStreamExt::error(&media_file).unwrap();
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: disambiguate the associated function for candidate #3
    |
138 |                     let error = gtk4::prelude::PrintOperationExt::error(&media_file).unwrap();
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: disambiguate the associated function for candidate #4
    |
138 |                     let error = gtk4::prelude::MediaStreamExtManual::error(&media_file).unwrap();
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/gtk4/src/media_stream.rs
+++ b/gtk4/src/media_stream.rs
@@ -7,11 +7,11 @@ use glib::IsA;
 pub trait MediaStreamExtManual: 'static {
     #[doc(alias = "gtk_media_stream_gerror")]
     #[doc(alias = "gtk_media_stream_error")]
-    fn error(&self, error: glib::Error);
+    fn set_error(&self, error: glib::Error);
 }
 
 impl<O: IsA<MediaStream>> MediaStreamExtManual for O {
-    fn error(&self, error: glib::Error) {
+    fn set_error(&self, error: glib::Error) {
         unsafe {
             ffi::gtk_media_stream_gerror(
                 self.as_ref().to_glib_none().0,

